.gitignore

```diff
@@ -25,13 +25,15 @@ test/feature-benchmark/tmp/*.td
 test/zippy/tmp/*.td
 test/**/tmp_*.toml
 junit_*.xml
+parallel-workload-queries.log
 perf.data*
 **/*.rej
 **/*.orig
 **/*.profraw
 **/*.profdata
 **/*.debug
 scratch
+services.log
```

Comment:
Do you happen to know where the logging path is specified? When running scalability tests from `tests/scalability`, I get the `services.log` file added to the repo root.

Reply:
Actually it's the opposite. `**/` prefix is useless. I removed it from all other .gitignore entries too.